src/packages/components/ResultCard.tsx

```diff
@@ -0,0 +1,45 @@
+import { Avatar, AvatarImage } from "@packages/shadcn-ui/ui/avatar";
+import { Card } from "@packages/shadcn-ui/ui/card";
+
+interface ElectionResult {
+  image: string;
+  candidate: string;
+  vice: string;
+  party: string;
+  percentagem: string;
+  votos: string;
+}
+function ResultCard({
+  candidate,
+  vice,
+  party,
+  percentagem,
+  votos,
+  image,
+}: ElectionResult) {
+  return (
+    <Card className="flex flex-row w-[60vw] p-4 shadow-md shadow-gray-200 border-[rgba(0,0,0,0)]">
+      <div className="flex w-[40vw] gap-4">
+        <Avatar className="w-16 h-16">
+          <AvatarImage src={image} />
+        </Avatar>
+        <div>
+          <p className="font-bold">Presidente: {candidate}</p>
```

Comment:
It's ok to have portuguese on the text displayed to the user (we can enhance that with internationalization) but code and variables, always in colonizador language